dashboard/views.py

```diff
@@ -175,45 +181,82 @@ class PasswordResetCompleteView(PasswordResetCompleteView):
 
 
 @staff_member_required
-def staff_inactive(request,id):
-	u = User.objects.get(pk=id)
-	u.is_active=False
-	u.save()
-	result=User.objects.all().exclude(id=request.user.id)
-	return render(request, 'staff1.html',{'result':result})
+def staff_inactive(request, id):
+    u = User.objects.get(pk=id)
+    u.is_active = False
+    u.save()
+    result = User.objects.all().exclude(id=request.user.id)
+    return render(request, 'staff_list.html', {'result':result})
 
 
 @staff_member_required
-def staff_active(request,id):
-	u = User.objects.get(pk=id)
-	u.is_active=True
-	u.save()
-	result=User.objects.all().exclude(id=request.user.id)
-	return render(request, 'staff1.html',{'result':result})
+def staff_active(request, id):
+    u = User.objects.get(pk=id)
+    u.is_active = True
+    u.save()
+    result = User.objects.all().exclude(id=request.user.id)
+    return render(request, 'staff_list.html', {'result':result})
 
 @staff_member_required
 def staff_add(request,id):
-	u = User.objects.get(pk=id)
-	u.is_staff=True
-	u.save()
-	result=User.objects.all().exclude(id=request.user.id)
-	return render(request, 'staff1.html',{'result':result})
+    u = User.objects.get(pk=id)
+    u.is_staff = True
+    u.save()
+    result = User.objects.all().exclude(id=request.user.id)
+    return render(request, 'staff_list.html', {'result':result})
 
 @staff_member_required
 def staff_remove(request,id):
-	u = User.objects.get(pk=id)
-	u.is_staff=False
-	u.save()
-	result=User.objects.all().exclude(id=request.user.id)
-	return render(request, 'staff1.html',{'result':result})
+    u = User.objects.get(pk=id)
+    u.is_staff = False
+    u.save()
+    result = User.objects.all().exclude(id=request.user.id)
+    return render(request, 'staff_list.html', {'result':result})
 
-@staff_member_required
-def staff_mem(request):
-	result=User.objects.all().exclude(id=request.user.id)
-	return render(request, 'staff1.html',{'result':result})
 
+class StaffList(LoginRequiredMixin, ListView):
+    model = User
+    template_name = 'staff_list.html'
+    paginate_by = 10
+    paginate_orphans=1
+    context_object_name = 'result'
 
+    @method_decorator(staff_member_required)
+    def dispatch(self, *args, **kwargs):
+        return super().dispatch(*args, **kwargs)
 
-	
-	
-	
\ No newline at end of file
+
+@staff_member_required
+def staff_edit(request, id):
+    data = get_object_or_404(User,id=id)
+    data = User.objects.get(id=id)
+    if request.method == "GET":
+        form = UserEditForm(instance=data)
+        print(form.errors)
+        return render(request,'staffedit.html', {'form':form})
+
+    if request.method == "POST":
+        form = UserEditForm(request.POST, request.FILES, instance=data)
+        if form.is_valid():
+            form.save()
+            result = User.objects.all().exclude(id=request.user.id)
```

Comment:
I think No need of `result = User.objects.all().exclude(id=request.user.id)`  because you are redirecting to other page.